myEnigma/myEnigma.ino

```diff
@@ -1461,9 +1461,9 @@ void setConfig(enigmaModel_t model,
 	       uint8_t rotor2,
 	       uint8_t rotor3,
 	       uint8_t etw,
-	       char    ringSetting[WALZECNT],	//=" AAA"
-	       char    plugboard[40],		//="AB CD EF GH IJ KL MN OP QR ST UV WX YZ",
-	       char    currentrotor[WALZECNT]	//=" AAA"
+	       const char    ringSetting[WALZECNT],	//=" AAA"
+	       char          plugboard[40],		//="AB CD EF GH IJ KL MN OP QR ST UV WX YZ",
+	       const char    currentrotor[WALZECNT]	//=" AAA"
```

Comment:
Need to test this my self. From what little I understand having something "const char bla[]" means I can not change the content of the array. (this goes for all the const changes)